### ISSUE TITLE
Game Crash

### ISSUE THREAD

**Issue by bluelionman** [triplea-game/triplea#3218]
### Engine version

1.9.0.0.3635

### My Operating System
Windows 7

### Map name
Big World 1942: v3 Rules

### Can you describe how to trigger the error? (eg: what sequence of actions will recreate it?)
I was in Round 20 as UK did naval and parachute landing on Japan and it crashed came up with below error info. it was last attack all other attacks that round executed fine. 

### Do you have the exact error text? Please copy/paste if so
```
triplea.engine.version.bin:1.9
Loading map: big_world, from: C:\Users\User\triplea\downloadedMaps\big_world-master.zip
Loading resources from the following paths: [C:\Users\User\triplea\downloadedMaps\big_world-master.zip, C:\Program Files\TripleA_1.9.0.0.3635\assets]
Loading map: big_world, from: C:\Users\User\triplea\downloadedMaps\big_world-master.zip
Loading resources from the following paths: [C:\Users\User\triplea\downloadedMaps\big_world-master.zip, C:\Program Files\TripleA_1.9.0.0.3635\assets]
17 - Americans. strafing territory: Southern Japan, alliedPlayer=British, maxWin%=97.61904761904762, maxAttackers=17, maxDefenders=15
18 - Americans. strafing territory: Southern Japan, alliedPlayer=British, maxWin%=92.5, maxAttackers=19, maxDefenders=20
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 71 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 71 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 72 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 71 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 71 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 72 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
java.lang.IllegalStateException: Not being carried, unit:infantry owned by British transport:bomber owned by British
	at games.strategy.triplea.delegate.TransportTracker.unloadAirTransportChange(TransportTracker.java:162)
	at games.strategy.triplea.delegate.MustFightBattle.landParatroops(MustFightBattle.java:2379)
	at games.strategy.triplea.delegate.MustFightBattle.access$700(MustFightBattle.java:50)
	at games.strategy.triplea.delegate.MustFightBattle$9.execute(MustFightBattle.java:758)
	at games.strategy.triplea.delegate.ExecutionStack.execute(ExecutionStack.java:35)
	at games.strategy.triplea.delegate.MustFightBattle.fight(MustFightBattle.java:392)
	at games.strategy.triplea.delegate.BattleDelegate.fightBattle(BattleDelegate.java:227)
	at sun.reflect.GeneratedMethodAccessor94.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.delegate.DelegateExecutionManager$1.invoke(DelegateExecutionManager.java:148)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor107.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeSingle(EndPoint.java:172)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeMultiple(EndPoint.java:150)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeLocal(EndPoint.java:132)
	at games.strategy.engine.message.unifiedmessenger.UnifiedMessenger.invokeAndWait(UnifiedMessenger.java:101)
	at games.strategy.engine.message.UnifiedInvocationHandler.invoke(UnifiedInvocationHandler.java:50)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor94.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.gamePlayer.GameOverInvocationHandler.invoke(DefaultPlayerBridge.java:162)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at games.strategy.triplea.TripleAPlayer.battle(TripleAPlayer.java:484)
	at games.strategy.triplea.TripleAPlayer.start(TripleAPlayer.java:131)
	at games.strategy.engine.framework.ServerGame.waitForPlayerToFinishStep(ServerGame.java:531)
	at games.strategy.engine.framework.ServerGame.runStep(ServerGame.java:424)
	at games.strategy.engine.framework.ServerGame.startGame(ServerGame.java:272)
	at games.strategy.engine.framework.startup.launcher.LocalLauncher.launchInNewThread(LocalLauncher.java:61)
	at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$117(AbstractLauncher.java:43)
	at java.lang.Thread.run(Unknown Source)
	at games.strategy.engine.message.UnifiedInvocationHandler.invoke(UnifiedInvocationHandler.java:62)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor94.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.gamePlayer.GameOverInvocationHandler.invoke(DefaultPlayerBridge.java:162)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at games.strategy.triplea.TripleAPlayer.battle(TripleAPlayer.java:484)
	at games.strategy.triplea.TripleAPlayer.start(TripleAPlayer.java:131)
	at games.strategy.engine.framework.ServerGame.waitForPlayerToFinishStep(ServerGame.java:531)
	at games.strategy.engine.framework.ServerGame.runStep(ServerGame.java:424)
	at games.strategy.engine.framework.ServerGame.startGame(ServerGame.java:272)
	at games.strategy.engine.framework.startup.launcher.LocalLauncher.launchInNewThread(LocalLauncher.java:61)
	at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$117(AbstractLauncher.java:43)
	at java.lang.Thread.run(Unknown Source)
```
### Instead of this error, what should have happened?
game should of played out

### Any additional information that may help

I made a zip file in my windows but cant see how to attach it here?
```
triplea.engine.version.bin:1.9
Loading map: big_world, from: C:\Users\User\triplea\downloadedMaps\big_world-master.zip
Loading resources from the following paths: [C:\Users\User\triplea\downloadedMaps\big_world-master.zip, C:\Program Files\TripleA_1.9.0.0.3635\assets]
Loading map: big_world, from: C:\Users\User\triplea\downloadedMaps\big_world-master.zip
Loading resources from the following paths: [C:\Users\User\triplea\downloadedMaps\big_world-master.zip, C:\Program Files\TripleA_1.9.0.0.3635\assets]
17 - Americans. strafing territory: Southern Japan, alliedPlayer=British, maxWin%=97.61904761904762, maxAttackers=17, maxDefenders=15
18 - Americans. strafing territory: Southern Japan, alliedPlayer=British, maxWin%=92.5, maxAttackers=19, maxDefenders=20
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 71 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 71 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 72 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 71 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 71 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
WARNING [Triplea start thread] ProAI ->       19-americanNonCombatMove: could not move [armour owned by British] over Route:SZ 72 South China Sea -> Borneo because: Player, Americans, is not owner of all the units: 1 armour
java.lang.IllegalStateException: Not being carried, unit:infantry owned by British transport:bomber owned by British
	at games.strategy.triplea.delegate.TransportTracker.unloadAirTransportChange(TransportTracker.java:162)
	at games.strategy.triplea.delegate.MustFightBattle.landParatroops(MustFightBattle.java:2379)
	at games.strategy.triplea.delegate.MustFightBattle.access$700(MustFightBattle.java:50)
	at games.strategy.triplea.delegate.MustFightBattle$9.execute(MustFightBattle.java:758)
	at games.strategy.triplea.delegate.ExecutionStack.execute(ExecutionStack.java:35)
	at games.strategy.triplea.delegate.MustFightBattle.fight(MustFightBattle.java:392)
	at games.strategy.triplea.delegate.BattleDelegate.fightBattle(BattleDelegate.java:227)
	at sun.reflect.GeneratedMethodAccessor94.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.delegate.DelegateExecutionManager$1.invoke(DelegateExecutionManager.java:148)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor107.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeSingle(EndPoint.java:172)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeMultiple(EndPoint.java:150)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeLocal(EndPoint.java:132)
	at games.strategy.engine.message.unifiedmessenger.UnifiedMessenger.invokeAndWait(UnifiedMessenger.java:101)
	at games.strategy.engine.message.UnifiedInvocationHandler.invoke(UnifiedInvocationHandler.java:50)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor94.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.gamePlayer.GameOverInvocationHandler.invoke(DefaultPlayerBridge.java:162)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at games.strategy.triplea.TripleAPlayer.battle(TripleAPlayer.java:484)
	at games.strategy.triplea.TripleAPlayer.start(TripleAPlayer.java:131)
	at games.strategy.engine.framework.ServerGame.waitForPlayerToFinishStep(ServerGame.java:531)
	at games.strategy.engine.framework.ServerGame.runStep(ServerGame.java:424)
	at games.strategy.engine.framework.ServerGame.startGame(ServerGame.java:272)
	at games.strategy.engine.framework.startup.launcher.LocalLauncher.launchInNewThread(LocalLauncher.java:61)
	at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$117(AbstractLauncher.java:43)
	at java.lang.Thread.run(Unknown Source)
	at games.strategy.engine.message.UnifiedInvocationHandler.invoke(UnifiedInvocationHandler.java:62)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor94.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at games.strategy.engine.gamePlayer.GameOverInvocationHandler.invoke(DefaultPlayerBridge.java:162)
	at com.sun.proxy.$Proxy7.fightBattle(Unknown Source)
	at games.strategy.triplea.TripleAPlayer.battle(TripleAPlayer.java:484)
	at games.strategy.triplea.TripleAPlayer.start(TripleAPlayer.java:131)
	at games.strategy.engine.framework.ServerGame.waitForPlayerToFinishStep(ServerGame.java:531)
	at games.strategy.engine.framework.ServerGame.runStep(ServerGame.java:424)
	at games.strategy.engine.framework.ServerGame.startGame(ServerGame.java:272)
	at games.strategy.engine.framework.startup.launcher.LocalLauncher.launchInNewThread(LocalLauncher.java:61)
	at games.strategy.engine.framework.startup.launcher.AbstractLauncher.lambda$launch$117(AbstractLauncher.java:43)
	at java.lang.Thread.run(Unknown Source)
```

**Comment by bluelionman** [comment]
[autosave2.zip](https://github.com/triplea-game/triplea/files/1777808/autosave2.zip)

Thats the game - you can load and play it but when you do the Japanese mainland battle it will crash every time :-1:

**Comment by RoiEXLab** [comment]
@bluelionman Thanks for your report.
Could you try updating to the [latest version of TripleA](http://triplea-game.org/download/) and check if this error persists?

**Comment by bluelionman** [comment]
How would I do that as I've been using game saves?

**Comment by RoiEXLab** [comment]
@bluelionman I'm not sure what exactly is your problem with updating 🤔 
Once you've downloaded the installer from the website, the savegame should open automatically in the engine when double clicking it, alternatively you can launch the newer version of the game and load in the savegame via the menu on the left side.

**Comment by bluelionman** [comment]
Okay I updated - loaded saved game and exacly the same thing happened during the mainliand Japan invasion. Crashed - got that error box come up?

**Comment by RoiEXLab** [comment]
@bluelionman Thanks for checking I just wanted to ensure this issue hasn't been fixed since then (the version you were using is probably over a year old already).

@ron-murhammer Could you have a look?

**Comment by ron-murhammer** [comment]
@bluelionman @RoiEXLab I'm pretty sure this is the same issue that was fixed in #2776. If you follow these steps it should resolve the problem:
1. Download the latest pre-release: https://github.com/triplea-game/triplea/releases/tag/1.9.0.0.9551
2. Take a save from before combat move
3. Redo the combat moves including the paradrop in Japan
